frontend/src/stores/profileStore.jsx

```diff
@@ -1,17 +1,13 @@
 import { create } from "zustand";
 const apiEnv = import.meta.env.VITE_BACKEND_API;
+import { userStore } from "../stores/userStore";
 
 export const profileStore = create((set) => ({
   // Set the initial data
-  lastName: "",
-  firstName: "",
-  phone: "",
-  important: "",
-  color: "",
-  flower: "",
+  profile: [],
```

Comment:
We can consider adding the spread operator like Veronica mentioned and like pinpointed in the video tutorial for keeping the previous values?
` // updating the user's profile
    set((state) => ({
     profile: [
        ...state.profile,
        {
          lastName,
          firstName,
          phone,
          important,
         color,
flower
        },
      ],
    }));`